store/postgres/src/chain_store.rs

```diff
@@ -617,12 +617,30 @@ mod data {
                     .optional()?,
             };
 
+            fn try_to_int(ts: Option<String>) -> Result<Option<u64>, StoreError> {
+                let ts = match ts {
+                    Some(str) => str,
+                    None => return Ok(None),
+                };
+
+                let radix = if ts.starts_with("0x") { 16 } else { 10 };
+
+                u64::from_str_radix(&ts[2..], radix)
```

Comment:
This isn't right when the string is base 10 - there we don't want to skip the first two chars